ui/components/app/collectibles-detection-notice/collectibles-detection-notice.js

```diff
@@ -20,7 +20,7 @@ export default function CollectiblesDetectionNotice() {
   const history = useHistory();
 
   return (
-    <Box marginBottom={8} className="collectibles-detection-notice">
+    <Box marginBottom={5} className="collectibles-detection-notice">
```

Comment:
Adhere to 8px grid use `4`(4x4px=16px) or `6`(6x4px=24px) here

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13291/commits/71a02813c79dbb632742095c0ea0566fd029eb39